site/write_ups/2023/25-01-23_writeup.md

```diff
@@ -0,0 +1,108 @@
+
+Zoë wants to interview the Data Ethics organisers on the NHS-R Community podcast that she started being involved with last year. She is planning to introduce/mention Data Hazards in workshops/meetings to raise awareness and try to tie it into everyday work that analysts in the NHS/Councils do. She also is going to research/read more about [Palantir](https://www.bbc.co.uk/news/business-54348456 and https://www.theguardian.com/society/2022/jun/21/palantir-concerns-over-data-firm-poised-to-be-operating-system-of-nhs) and the possible implications for the work at the National level with their data platforms, particularly linked to the recent announcement of working with Posit (formerly RStudio). This might be something that could be discussed in an ethics committee type format.
+
+Alessia plans to pilot similar work at Wellcome Genome Campus to facilitate data scientists (and other professionals) to engage with ethical issues relevant to their work! 
+
+Euan wants to find ways to introduce more Data Ethics material (and memes, of course) in lectures/tutorials for students and staff at his work. Euan also wants to successfully navigate academia collaboratively, rejecting the competition paradigm (slightly more ambitious than remembering to say "please" and "thank you" to your at home Google assistant!)
+
+Our very own Nina wants to find a way to spread the word about data ethics to other fields using data (e.g. psychology! medicine!) - this might be by running a data hazards workshop. She would also like to find a way to make this formally part of my job, rather than just a passion project. Nina would also like to be brave enough to engage more with the data ethics community, maybe by attending a conference?
+
+Vanessa wants to do a data ethics project with non-technical people, publish her data ethics work and make masters students contribute to her data ethics research.
+
+Jess wants to complete applied data science module and learn more about the technical side as well as get involved helping out with data ethics club!
+
+Ola wants to learn some tech/data visualisation skills to become a more well rounded person - Huw suggests the [JGI Data Viz seminar](https://dataviz.blogs.bristol.ac.uk/)! Ola will run a Digital Societies Reading group (late Feb-ish) on ground truth in ML: Kang, E. B. (2023), Ground truth tracings (GTT): On the epistemic limits of machine learning and Big Data & Society, 10(1). Precise details to be confirmed but everyone is invited so stay tuned!
+
+Noshin would like to think more about intersectionality in data and from whom's perspective is the data story being told. Holding the experience behind the data in mind when reporting on key themes surrounding children's social care. Also we don't do much peer review for our thematics, so it would be good to have this challenge! 
```

Comment:
```suggestion
Noshin would like to think more about intersectionality in data and from whom's perspective is the data story being told. Holding the experience behind the data in mind when reporting on key themes surrounding children's social care. Also there isn't much peer review in her context, so it would be good to have this challenge! 
```